zomes/trust_atom/coordination/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "hc_zome_tg_coordination"
```

Comment:
I’m thinking a better name is “trust_atom” — looking at it from the perspective of the importing code.  

Maybe then the other one is “trust_atom_integrity”

Thoughts?

Reply:
yea that sounds good